node/node-core/src/eth/transaction/ethers_compat.rs

```diff
@@ -0,0 +1,161 @@
+//! ethers compatibility, this is mainly necessary so we can use all of `ethers` signers
```

Comment:
Can we instead PR any necessary changes to Ethers, instead of creating new types?

Reply:
that would be ideal, but was very confused by how rlp is implemented there, can have closer look